packages/php-wasm/universal/src/lib/wasm-error-reporting.ts

```diff
@@ -34,14 +34,14 @@ export class UnhandledRejectionsTarget extends EventTarget {
  * @param runtime
  */
 export function improveWASMErrorReporting(runtime: Runtime) {
-	runtime.asm = {
-		...runtime.asm,
+	runtime.wasmExports = {
+		...runtime.wasmExports,
```

Comment:
I originally thought that this part wasn't necessary but found [this commit](https://github.com/WordPress/wordpress-playground/commit/15a6e75e38abb4e7c67e1b8f665be8420b279fbf) that made me think it was important to have it. Curiously enough, before adding this part, unit tests succeeded locally as shared in [this comment](https://github.com/WordPress/wordpress-playground/pull/1471#issuecomment-2152724442). After re-introducing it, unit tests began to fail again with [the same error](https://github.com/WordPress/wordpress-playground/pull/1471#issuecomment-2147161808).

In order to address the failures in unit tests, I'm going to remove it. cc @adamziel as you introduced the changes [here](https://github.com/WordPress/wordpress-playground/commit/15a6e75e38abb4e7c67e1b8f665be8420b279fbf), in case you have any insights. Thanks 🙇 !

Reply:
Changed applied in https://github.com/WordPress/wordpress-playground/pull/1471/commits/c6a352a1d63d332e566b61c58c9607182767eba4.